### PR TITLE
Create update for Muni Boundries

### PR DESCRIPTION
**Pull request by mheagin**
Newest updates for Muni Boundaries

### CODE REVIEW

**Comment by steveoh** [comment]
Anna is going to take a crack at this tomorrow and we'll merge it then!

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - SGID Blog
+tags:
+  - city
+  - town
+  - municipalities
+  - boundaries
+  - annexations
+---
+
+It’s already over three-quarters of the way through the year, things have slowed down while everyone is enjoying the summer, yet we’re still trying to catch up here. We still expect annexations to pick up some as we head for the end of the year, plenty of procrastinators out there. Here is our updates for the third quarter.
```
[the remark this answers is not in the document]
Comment (atribolet) [reply _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]: I never really know how to start out articles! I tried.

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+As always these changes to town and city boundaries can affect other departments too, be sure to share your data with others.
```
Comment (atribolet) [inline _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]:
```suggestion
As always, these changes to town and city boundaries can affect other boundary layers, so be sure to share your data with others.
```

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+We still want everyone to remember the 2020 census is coming up we are still pushing to get everything as perfect as we can so take a look at the state’s data and see if we are missing anything or if there is something strange, and let us know.
```
Comment (atribolet) [inline _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]:
```suggestion
First, we want to remind everyone that the 2020 census is coming up, and we are still pushing to get the State's data layers as perfect as we can. So please take a look at the State’s data and see whether we are missing anything, or if there is something strange, let us know.
```

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Here is a [census flyer](https://www2.census.gov/geo/pdfs/partnerships/GEO_Program_Flyer.pdf) giving reasons why GIS data is so important to the process.
```
Comment (atribolet) [inline _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]:
```suggestion
For information about why GIS data is so important to the census process, check out this [census flyer](https://www2.census.gov/geo/pdfs/partnerships/GEO_Program_Flyer.pdf).
```

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md (hunk 3 above)
Comment (atribolet) [reply _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]: Maybe we should add a "Reminder" or "Census 2020" heading? Also, maybe we should move the census info before the reminder about changes to town/city boundaries affecting other data layers? I feel like that should go closer to the actual updates.

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Municipalities and updates can be viewed on this [Utah Municipality Modifications](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63) web map.
```
Comment (atribolet) [inline _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]:
```suggestion
Municipalities and updates can be viewed on the [Utah Municipality Modifications](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63) web map.
```

Comment (atribolet) [reply _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]: Again, maybe add a heading with "Updates" or something generic?

File: _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Following are the changes and annexations since June 30, 2019, through Oct 15, 2019 listed by County and City:
```
Comment (atribolet) [inline _posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md]:
```suggestion
The following table shows the changes and annexations (listed by county and city) from June 30, 2019, to October 15, 2019:
```

**Comment by atribolet** [comment]
(Should I have clicked "request changes" instead of "approve"? I need to make note of this for future reference!)

**Comment by steveoh** [comment]
@atribolet you can click whatever you want really. Approve if there are no changes, a comment is fine when you request changes or you can specifically say these changes should happen. It's sorta up to you.

**Pull request merged**